bin/kafka-run-class.sh

```diff
@@ -111,7 +111,7 @@ else
   for file in "$base_dir"/streams/upgrade-system-tests-$SHORT_VERSION_NO_DOTS/build/libs/kafka-streams-upgrade-system-tests*.jar;
   do
     if should_include_file "$file"; then
-      CLASSPATH="$CLASSPATH":"$file"
+      CLASSPATH="$file":"$CLASSPATH"
```

Comment:
why this?

Reply:
For the same reason in #5605, in enabling the upgrade tests the trunk version of `StreamsUpgradeTest` was executed over the version in `upgrade-system-test` modules. As for why it wasn't a problem before, I can't say.